packages/beacon-node/src/chain/blocks/verifyBlocksExecutionPayloads.ts

```diff
@@ -319,7 +319,7 @@ export async function verifyBlockExecutionPayload(
       const lvhResponse = {
         executionStatus,
         latestValidExecHash: execResult.latestValidHash,
-        invalidateFromBlockHash: toHexString(block.message.parentRoot),
+        invalidateFromParentBlockHash: toHexString(block.message.parentRoot),
```

Comment:
I know this is just adding "parent" to this variable name, but we should either call this "parent root" or "parent block root" instead of "parent block hash".